Packs/SecureWorks/Integrations/TaegisXDRv2/TaegisXDRv2.py

```diff
@@ -1203,8 +1242,8 @@ def fetch_investigation_command(client: Client, env: str, args=None):
         }
         """ % (fields)
         variables = {
-            "page": args.get("page", 0),
-            "perPage": args.get("page_size", 10),
+            "page": arg_to_number(args.get("page")) or 0,
```

Comment:
You can still use arg_to_number(args.get("page", 0))
arg_to_number can recieve int or string and will return an int

Reply:
This format was used mostly for linting/typing issues. However, I went head and added a `# type: ignore` comment to locations where this might be a problem and updated all locations where this format was in use.